src/test/java/org/bankofspring/BankServiceTest.java

```diff
@@ -0,0 +1,136 @@
+package org.bankofspring;
+
+import static org.junit.Assert.*;
+
+import org.bankofspring.model.Account;
+import org.bankofspring.model.Customer;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("classpath:BankOfSpringApp.xml")
+
+public class BankServiceTest {
+
+	@Autowired
+	private BankService service;
+	
+	@Autowired
+	@Qualifier("customer1")
+	private Customer customer1;
+	
+	@Autowired
+	@Qualifier("account1")
+	private Account account1;
+	
+	@Autowired
+	@Qualifier("account2")
+	private Account account2;
+	
+	
+	@Test
+	@DirtiesContext //force reload of context (otherwise amount for account changes)
```

Comment:
Just, had a look, you could have added it at class level so you didn't have to copy it for each test. Still good though.